fluent-react/src/localized.ts

```diff
@@ -66,8 +66,9 @@ export function Localized(props: LocalizedProps): ReactElement {
   }
 
   if (!l10n) {
```

Comment:
This branch previously would never be hit, as `l10n` always existed. The change to `fluent-react/src/context.ts` means that this branch can now be exercised.